Dockerfile

```diff
@@ -14,12 +14,15 @@ CMD ["/app/entrypoint.sh", "start"]
 ENV REPOSITORY_LOCAL_PATH=/repo
 ENV TMPDIR=/tmp
 
+ARG GIT_VERSION=2.49.0
+
 USER root
 
 # Runtime dependencies
 RUN apk --no-cache --update add \
     composer \
     curl \
+    expat \
     freetype \
     g++ \
     git \
```

Comment:
Should we remove this `git` install if we're installing it from source below?

Is there are way we could get the latest version from a package manger?